app/assets/v2/js/grants/new.js

```diff
@@ -71,64 +71,39 @@ $(document).ready(function() {
           SubscriptionContract.deploy({
             data: compiledSubscription.bytecode,
             arguments: args
-          })
-            .send({
-              from: accounts[0],
-              gas: 2500000,
-              gasPrice: 4000000000
-            })
-            .on('error', function(error) {
-              console.log('1', error);
-            })
-            .on('transactionHash', function(transactionHash) {
-              console.log('2', transactionHash);
-              $('#transaction_hash').val(transactionHash);
-
-              // Waiting State screen
-              $('#new-grant').hide();
-              $('.interior .body').addClass('open');
-              $('.interior .body').addClass('loading');
-              $('.grant_waiting').show();
-              document.issueURL = $('#input-url').val();
-              waitingStateActive();
-
-            })
-            .on('receipt', function(receipt) {
-              $('#receipt').val(JSON.stringify(receipt));
-              $('#contract_address').val(receipt.contractAddress);
-            })
-            .then(function(contractInstance) {
-
-              console.log(contractInstance);
-
-              $.each($(form).serializeArray(), function() {
-                data[this.name] = this.value;
-              });
-              console.log(data);
-              form.submit();
+          }).send({
+            from: accounts[0],
+            gas: 2500000,
+            gasPrice: 4000000000
```

Comment:
why the hardcoded gas price?  does this pattern exist all across grants?  there is lots of good gas price info in the backend and in the gas price module.

Reply:
I just re-indented that ! 😅 
@captnseagraves would you be able to answer this ?

Reply:
@owocki @thelostone-mc Hardcoded gasPrice because there is a frequent error on Rinkeby when the gasPrice is not set and defaults to 1 gwei. Implementing the gasPrice functionality already within Bounties has not been a priority up to now, so has yet to be done. The above only prefills the value in metamask and still allows the user to adjust as they see fit.